utils/data.py

```diff
@@ -535,46 +621,76 @@ def fload_and_patch(self, selectedPatients, selectedDatasets):
                     else:
                         # concatenate all patches in one array
                         if self.patchingMode == 'PATCHING_2D':
-                            dAllPatches = np.concatenate((dAllPatches, dPatches), axis=2)
-                            dAllLabels = np.concatenate((dAllLabels, dLabels), axis=0)
-                            dAllPats = np.concatenate((dAllPats, dPats), axis=0)
+                            dAllPatches = np.concatenate((dPatches,dPatches_phase ), axis=2)
```

Comment:
this is loosing the concatenation of all patches

Reply:
this also heavily determines the way you read in the patches: 
a) as separate channel
b) first magnitude, then phase, then magnitude, then phase, ....